package.json

```diff
@@ -9,7 +9,8 @@
     "doclint": "npm run doc -- -t templates/silent",
     "test": "npm run lint && npm run build && npm run doclint",
     "build": "webpack -p",
```

Comment:
Yes, I couldn't find any clear documentation of what happens when within NPM lifecycles, and was mislead by some documentation.